.github/turbo-orchestrator.yml

```diff
@@ -38,6 +38,14 @@ labeler:
       when:
         isAnyFileOwnedByMatch: '@vercel\/web-tooling'
 
+    # created-by
+    - label: "created-by: web-tooling team"
+      when:
+        isPRAuthorMatch: "^(alexkirsz|Brooooooklyn|ForsakenHarmony|jridgewell|kdy1|kwonoj|padmaia|sokra|wbinnssmith)$"
+    - label: "created-by: turborepo team"
+      when:
+        isPRAuthorMatch: "^(gsoltis|gaspar09|nathanhammond|tknickman|mehulkar|chris-olszewski|NicholasLYang|arlyon)$"
```

Comment:
Eventually we should be able to dynamically source member lists from our Github teams, but this will work for now.